components/LoginModal.vue

```diff
@@ -288,4 +287,32 @@ export default {
   display: flex;
   justify-content: center;
 }
+::v-deep .sf-icon-path {
+  margin: 0;
+}
+.login-button {
+  margin: 0 0 0 6.875rem;
+  border-radius: 0.25rem;
+  width: 11.625rem;
+  height: 2.625rem;
+  font-size: 1.125rem;
+  padding: 0.625rem 0 0 0;
```

Comment:
for margin, height, width and padding use spacer instead of rem

Reply:
done